fuzzy-search/info.json

```diff
@@ -57,12 +48,91 @@
                     "visible": true,
                     "editable": true,
                     "value": 75,
-                    "description": "Specify threshold to match key value data in JSON.",
-                    "tooltip": "Specify threshold to match key value data in JSON."
+                    "tooltip": "Specify the similarity threshold (percentage) to match key value data in JSON.",
+                    "description": "Specify the similarity threshold (percentage) to match key value data in JSON."
+                },
+                {
+                    "title": "Fuzzy Match",
+                    "type": "checkbox",
+                    "name": "fuzzyMatch",
+                    "required": false,
+                    "visible": true,
+                    "editable": true,
+                    "value": true,
+                    "tooltip": "Check to filter data for fuzzy format.",
```

Comment:
Suggestion: Select to filter data using fuzzy logic

Reply:
Incorporated